app/common/model-service.js

```diff
@@ -66,6 +66,9 @@ angular.module('jamstash.model', ['jamstash.utils'])
         if (typeof album.coverArt != 'undefined') {
             coverartthumb = globals.BaseURL() + '/getCoverArt.view?' + globals.BaseParams() + '&size=160&id=' + album.coverArt;
             coverartfull = globals.BaseURL() + '/getCoverArt.view?' + globals.BaseParams() + '&id=' + album.coverArt;
+        } else {
+            coverartthumb = 'images/albumdefault_60.jpg';
```

Comment:
This should be `images/albumdefault_160.jpg` to match the `size=160` param 3 lines up.